mflowgen/glb_tile/custom-init/outputs/edge-blockages.tcl

```diff
@@ -0,0 +1,25 @@
+#=========================================================================
+# edge-blockages.tcl
```

Comment:
Should this be a single script in one common directory, as opposed to several identical scripts in different places?